docker/bootstrap/Dockerfile.common

```diff
@@ -1,4 +1,4 @@
-FROM --platform=linux/amd64 golang:1.23.4-bullseye
+FROM --platform=linux/amd64 golang:1.23.4-bookworm
 
 # Install Vitess build dependencies
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
```

Comment:
`etcd` package do not longer exists in bookworm if I am not mistaken. Should be replaced with `etcd-client` and `etcd-server`